guides-and-tutorials/sbtc/how-to-run-sbtc-signer.md

```diff
@@ -0,0 +1,531 @@
+# How to Run a sBTC Signer
+
+{% hint style="warning" %}
+This documentation is for preview purposes only. The implementation is not final and should not be used in production. The configuration files, especially the docker-compose setup, will undergo significant changes. This guide is provided to give potential operators an early look at the setup process.
+{% endhint %}
```

Comment:
Can we make this a bit less scary? The setup _might_ change, but we don't expect it to be completely different.

Reply:
```suggestion
{% hint style="warning" %}
This documentation is for preview purposes only. This should not be used in production. The configuration files and the docker-compose setup will undergo some necessary changes before release, but the setup here is very close to what will be required in production.
{% endhint %}
```